macaw-semmc/src/Data/Macaw/SemMC/TH/Monad.hs

```diff
@@ -19,14 +19,27 @@ module Data.Macaw.SemMC.TH.Monad (
   letBindExpr,
   letBindPureExpr,
   bindTH,
+  letBindPat,
   letTH,
   extractBound,
   refBinding,
   inConditionalContext,
```

Comment:
Is this now unused? I think I saw the only use I recalled being removed earlier

Reply:
No, this is used in ARM.Semantics.TH to destruct a tuple in the `StructField` case of `armAppEvaluator`